python/dasclient/dasclient.py

```diff
@@ -120,7 +121,10 @@ def _get(self, path, stream=False, **kwargs):
         headers = {'User-Agent': self.user_agent}
 
         headers.update(self.auth_headers())
-        response = requests.get(self._das_url(path), headers=headers, params=kwargs.get('params'), stream = stream)
+        if(not(path.startswith("http"))):
```

Comment:
don't need parens since we aren't breaking the line and it's not needed for clarity.
if not path.startswith("http"):